test/e2e/benchmarks_test.go

```diff
@@ -192,7 +192,7 @@ var _ = Describe("Podman Benchmark Suite", func() {
 			needsRegistry: true,
 			init: func() {
 				port, user, pass := getPortUserPass()
-				session := podmanTest.Podman([]string{"push", "--tls-verify=false", "--creds", user + ":" + pass, UBI_INIT, "localhost:" + port + "/repo/image:tag"})
```

Comment:
Be sure to go to the gear at top and select 'Hide whitespace' (if you use github gui. If you pull, use `git diff -w`).